purchase.adoc

```diff
@@ -388,288 +517,533 @@ Fields named "*UUID1" are the standard UUID Type 1 (8-4-4-4-12) representation o
+  "refunded": false,
   "groupedVatAmounts": {
-    "12.0": 1500
+    "25.0": 70000
   },
-  "refunded": false
+  "refund": false
 }
-```
-
-### Products
+----
+--
 
-A purchase has one or more item lines represented in the `products` list attribute. The `type` attribute indicates what
-kind of item line the entity represent, with `PRODUCT`, `CUSTOM_AMOUNT` and `GIFTCARD` being the currently available types.
+'''
+====
 
-In parallell to the `type` attribute there is an optional `details` attribute that may contain information related to the specific type,
-for example item lines of type `GIFTCARD` will have a `giftcardUuid` attribute pointing out what giftcard that was sold/returned as part of purchases
+==== Response Model
 
-### Payments
+===== Purchase
+Purchase represents a receipt, it's immutable and contains information about items that were purchased, one or several payments received, discounts applied etc.
+Refunds are represented by the same model but with negative amounts and refund related flags/fields. +
+*Note*: More attributes can potentially be added, please make sure your integration is build in a tolerant way.
 
-A purchase has one or more payments represented in the `payments` list attribute.
+*Attributes*
+[cols="15%,10%,75%"]
+|===
 
-=== Payment attributes
-The `payment` data type only contain a few generic parameters such as `uuid`, `amount` and `type`.
+|Name|Type|Description
 
-The currently available payment types are:
+|source|string a|Source of purchase, value can be one of the following
 
-- `IZETTLE_CARD`
-- `IZETTLE_CARD_ONLINE`
-- `IZETTLE_CASH`
-- `IZETTLE_INVOICE`
-- `SWISH`
-- `VIPPS`
-- `MOBILE_PAY`
-- `PAYPAL`
-- `STORE_CREDIT`
-- `GIFTCARD`
-- `KLARNA`
+* "POS" - purchase made with iZettle GO app
+* "WEB_SHOP" - purchase made online
+|purchaseUUID|string|Unique id of a purchase within iZettle system.
+|purchaseUUID1|string|Unique id of a purchase within iZettle system as UUID version 1.
+|timestamp|string|Created timestamp according to the https://en.wikipedia.org/wiki/ISO_8601[ISO 8601] format, e.g. "2019-12-03T17:05:06.123+0000".
+|purchaseNumber|integer|Incremental receipt number.
+|globalPurchaseNumber|integer|Incremental receipt number, only different from `purchaseNumber` if several cash registers are used by the merchant.
+|amount|integer|Gross amount (inc VAT) in minor currency units e.g  €9.95 EUR, amount will be 995. Is negative in case of refund.
+|vatAmount|integer|VAT amount for the whole purchase in minor currency units e.g €5 EUR, amount will be 500.
+|country|string|Two-letter country code for country the merchant operates in. Examples "SE", "NO", "GB", "FR", "DE", "DK".
+|currency|string|Currency code https://en.wikipedia.org/wiki/ISO_4217[ISO 4217]. Examples "SEK", "NOK", "GBP", "EUR".
+|products|array|Describes items that were sold. See section "Products" for more information.
+|payments|array|Payments that were processed/received. See section "Payments" for more information.
```

Comment:
Is it possible to add cross-references for section "Payments"?

Reply:
Well we need to make it a section first